praktikumsplaner-frontend/frontend/src/views/MeldungPraktikumsstelle/MeldungStudium.vue

```diff
@@ -163,14 +163,27 @@
                 </v-col>
                 <v-col cols="2" />
                 <v-col />
+                <v-col>
+                    <v-select
+                        v-model="praktikumsstelle.planstelleVorhanden"
+                        label="Planstelle"
```

Comment:
Since you have set the rule at the bottom, although in the wrong place, but you wanted to set it, I assume that it should be a mandatory field, which is why I would add an "*" at the end.